cypress/integration/score_card_spec.js

```diff
@@ -2,7 +2,7 @@ describe('Scorecard user flow', () => {
 
   beforeEach(() => {
     cy.visit('http://localhost:3000/')
-      .get('.search-btn').click().click()
+      .get('.search-btn').click()
   });
 
   it('should be able to add players to a round', () => {
```

Comment:
We should change this to describe end of round testing instead of adding players to the round. Or at least split up the tests.